### ISSUE TITLE
Error when installing package

### ISSUE THREAD

**Issue by goowikns** [jeroendesloovere/vcard#117]
Hi, 

When installing your package I receive following error:

  [Composer\Repository\RepositorySecurityException]
  The contents of https://packagist.org/p/provider-latest%240cbfb40ab72a881d2
  1b70f78286d39cd72e3b0eb8704c13e79dc49624e549973.json do not match its signa
  ture. This should indicate a man-in-the-middle attack. Try running composer
   again and report this if you think it is a mistake.

I can't seem to move on after this point :(

**Comment by jeroendesloovere** [comment]
Hi @goowikns,
thanks a lot for mentioning this.
I just tested `composer require jeroendesloovere/vcard` and indeed.
I'm getting the same error.

![schermafbeelding 2018-01-16 om 15 52 53](https://user-images.githubusercontent.com/588616/34994935-52de0cae-fad5-11e7-80e7-e0a59f31f9a7.png)

Very strange...

I just released a new version 1.5.2, in the hope of fixing this. But it did not fix the problem...
Probably a packagist bug?
If somebody knows the problem, any help is appreciated.
